llama_stack/apis/scoring_functions/scoring_functions.py

```diff
@@ -57,14 +57,14 @@ class ScoringFunctionDefWithProvider(ScoringFunctionDef):
 @runtime_checkable
 class ScoringFunctions(Protocol):
     @webmethod(route="/scoring_functions/list", method="GET")
-    async def list_scoring_functions(self) -> List[ScoringFunctionDefWithProvider]: ...
+    async def list_scoring_functions(self) -> List[ScoringFnDefWithProvider]: ...
```

Comment:
@raghotham Renamed the inner implementation names, do we also want to change these client facing interface to `list_scoring_functions` -> `list_scoring_fns` ?